bundles/action/org.openhab.action.slack/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,40 @@
+Manifest-Version: 1.0
+Private-Package: org.openhab.action.slack.internal
+Ignore-Package: org.openhab.action.slack.internal
+Bundle-License: http://www.eclipse.org/legal/epl-v10.html
+Bundle-Name: openHAB Slack Action
+Bundle-SymbolicName: org.openhab.action.slack
+Bundle-Version: 1.10.0.qualifier
+Bundle-Activator: org.openhab.action.slack.internal.SlackActivator
+Bundle-ManifestVersion: 2
+Bundle-Description: This is the Slack action of the open Home Autom
+ mation Bus (openHAB)
+Import-Package: org.apache.commons.io,
+ org.apache.commons.lang,
+ org.openhab.core.items,
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.scriptengine.action,
+ org.openhab.core.types,
+ org.osgi.framework,
+ org.osgi.service.cm,
+ org.osgi.service.component,
+ org.slf4j
+Bundle-DocURL: http://www.openhab.org
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Service-Component: OSGI-INF/action.xml
+Bundle-ClassPath: .,
+ lib/commons-logging-1.2.jar,
+ lib/gson-2.6.2.jar,
+ lib/httpclient-4.4.jar,
```

Comment:
Do you really need all those libraries in those specific versions? If not, you could try to work with the ones provided from the ESH core.

I am only using openHAB 2.x, so I don't know what list of libraries are provide by 1.x though 😊